js/services/MeteorService.js

```diff
@@ -0,0 +1,40 @@
+const dateUtils = require('../utils/DateUtils.js')
```

Comment:
You don't need to use .js in require section

Reply:
Fixed